apis/python/src/tiledbsoma/query_condition.py

```diff
@@ -91,20 +91,20 @@ class QueryCondition:
 
         ``val ::= <num> | <str> | val(val)``
 
-    **Example:**
-
-    >>> with tiledb.open(uri, mode="r") as A:
-    >>>     # Select cells where the attribute values for `foo` are less than 5
-    >>>     # and `bar` equal to string "asdf".
-    >>>     # Note precedence is equivalent to:
-    >>>     # tiledbsoma.QueryCondition("foo > 5 or ('asdf' == attr('b a r') and baz <= val(1.0))")
-    >>>     A.query(cond="foo > 5 or 'asdf' == attr('b a r') and baz <= val(1.0)")
-    >>>
-    >>>     # Select cells where the attribute values for `foo` are equal to
-    >>>     # 1, 2, or 3.
-    >>>     # Note this is equivalent to:
-    >>>     # tiledbsoma.QueryCondition("foo == 1 or foo == 2 or foo == 3")
-    >>>     A.query(cond="foo in [1, 2, 3]")
+    Example::
```

Comment:
double colon? TIL! :)

Reply:
It's a Sphinx directive to treat the next indented block like Python code with highlighting and everything.